File: java/FizzBuzz.java
public static void main(String args[]) {
    int length = 0;

    for (int i = 1; i < length; i++) {
        if (i % 15 == 0) System.out.println("FizzBuzz");
        else if (i % 5 == 0) System.out.println("Fizz");
        else if (i % 3 == 0) System.out.println("Buzz");
        else System.out.println(i);
    }
}

File: python/fizzbuzz.py
LENGTH = 30

for i in range(1, LENGTH):
    if i % 15 == 0: 
        print("FizzBuzz")
    elif i % 5 == 0: 
        print("Fizz")
    elif i % 5 == 0: 
        print("Buzz")
    else: 
        print(i)